openstack/backup-etcd/values.yaml

```diff
@@ -1,3 +1,10 @@
+owner-info:
+  maintainers:
+    - Falk Reimann
+    - Stefan Majewsky
+    - Sandro Jäckel
+    - Muhammad Talal Anwar
+
```

Comment:
backup-etcd has not been used for several years (ever since we moved the controlplane to Kubernikus). @reimannf Any objections to deleting this chart entirely?

Reply:
Yes it can be deleted

Reply:
Deleted ☠️